code_challenges/code_wars/trigrams.py

```diff
@@ -0,0 +1,17 @@
+def trigrams(phrase):
+    i = 0
+    j = 3
+    trigrams = []
+    updated_phrase = phrase.replace(" ", "_")
+    while i < len(phrase):
+        for character in updated_phrase:
+            
+            trigrams.append(updated_phrase[i:j])
+            i += 1
+            j += 1
+
+    return " ".join([trigram for trigram in trigrams if len(trigram) == 3])
+
```

Comment:
Per pep-008 https://peps.python.org/pep-0008/#blank-lines 2 blank lines surrounding functions in the global scope